build/configs/rtl8730e/hello_8M/defconfig

```diff
@@ -283,7 +283,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="48,52,12,400,3068,3068,512,1024,8,"
+CONFIG_FLASH_PART_SIZE="48,52,12,400,2940,2940,512,1280,8,"
```

Comment:
Because 16M is enough to give 256K more, but 8M isn't. Let's keep original size for 8M.